_events/2024/2024-05-education-training-talk.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Building and Running Containers on HPC Systems - A (not at all) Comprehensive Talk
+subtitle: US-RSE Education & Training Seminar Series
+expires: 2024-05-10
```

Comment:
I never know, does this need to be a day after the event or the day of? Will this expire at the end or at the beginning of the day?

Reply:
I just used the same format as one of the others. Hopefully it is correct.